scripts/serve.js

```diff
@@ -39,19 +39,16 @@ const packages = fs.readdirSync(packagesPath);
 
 const app = express();
 
-packages.forEach((name) => {
+packages.forEach(name => {
   const root = path.join(packagesPath, name);
   const entry = path.join(root, 'index.js');
   const dest = path.join(root, `${name}.bundle.js`);
-  const external = (id) => /^@(annotator|hot)/.test(id);
+  const external = id => /^@(annotator|hot)/.test(id);
```

Comment:
This is how prettier formats arrow functions with a single argument. There is [no option](https://github.com/prettier/prettier#options) to change this.

By design, prettier provides only a handful of options to choose major style options such as single vs. double quotes.